lib/member_name.rb

```diff
@@ -20,7 +20,7 @@ class MemberName < Scraped::HTML
 
   FEMALE_PREFIXES  = %w[mrs ms].freeze
   MALE_PREFIXES    = %w[mr prince].freeze
-  OTHER_PREFIXES   = %w[adv dr professor rev].freeze
+  OTHER_PREFIXES   = %w[adv dr professor rev prof].freeze
```

Comment:
Tiny nit: all the other examples here are in alphabetical order, so it would be good to maintain that.